docs/UserGuide.md

```diff
@@ -308,8 +313,39 @@ Tasks:
 
 ### 4.2 Applicant Features
 
+These are all the features in regard to an applicant in CMC.
+
 #### 4.2.1 Adding an applicant: `addApplicant` or `adda`
 
+Adds an applicant to the list of applicants in CMC.
+
+- Use `addApplicant` or `adda` to use add an applicant
+
+#### Usage:
+
+`addApplicant /name {applicantName} /phone {phoneNumber}`
+
+`adda /name {applicantName} /phone {phoneNumber}`
+
+#### Acceptable values for parameters:
+
+<div markdown="span" class="alert alert-primary">
+:information_source: **Notes about duplicate applicants:**
+
+Duplicate applicants are not allowed to be added into the list of applicants.
+
+An applicant is considered a duplicate if the applicant's name **or** phone number matches that of an existing applicant.
+
+This means that only an applicant with a unique name and phone number can be added into the list of applicants.
+</div>
+
+- `applicantName`: Only alphabetical characters, @, () are allowed
+- `phoneNumber`: Only numbers are allowed
+
+#### Example of usage:
+
+`adda /name John Doe /phone 92345678`
```

Comment:
![image](https://github.com/AY2324S1-CS2103T-W15-3/tp/assets/120372506/dbbd8b55-00c6-4bfb-a9a0-bd26d6ff3022)

I think can follow this image, add bullet point and description of what this example does. Also might need to add the addApplicant.png image below this